test/unit/tool_util/test_tool_linters_general.py

```diff
@@ -0,0 +1,60 @@
+from galaxy.tool_util.linters import general
```

Comment:
I'm not sure if we should have `test_*.py` files that don't actually contain tests.

Reply:
Should I rename the files or just put them back into one. The file seemed to big for me ..

Reply:
It doesn't seem to be particularly large compared to the other unit test files.